packages/node/src/filesystem/DynamicFilesystemRuntimeModule.js

```diff
@@ -0,0 +1,34 @@
+const { RuntimeModule, RuntimeGlobals, Template } = require('webpack');
+import {
+  fileSystemRunInContextStrategy,
+  httpEvalStrategy,
+  httpVmStrategy,
+} from './stratagies';
+
+class FileSystemRunInContextStrategyRuntimeModule extends RuntimeModule {
+  constructor() {
+    super('dynamic-file-system', 10);
```

Comment:
not important, but may use webpack.RuntimeModule.STAGE_BASIC is better than magic number

Reply:
Awesome, will do - thanks. What does the number prepresent in webpack? is this ordering of runtime module within the webpack runtime chunk?

Reply:
yes, the number determine the webpack runtime load order which the number bigger, the runtime load later